package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node server.js",
     "clean": "rimraf dist",
     "build": "npm run clean && npm run build:webpack",
+    "build:dev": "webpack --config webpack.config.js",
```

Comment:
There is a better analyse tool than the default one.
Next we should split into several files and not rebuild the lib ones, highcharts change once in a month or even less.